virt-v2v/cold/BUILD.bazel

```diff
@@ -89,15 +90,29 @@ container_image(
     visibility = ["//visibility:public"],
 )
 
+go_library(
+    name = "cold_lib",
+    srcs = ["entrypoint.go"],
+    importpath = "github.com/konveyor/forklift-controller/virt-v2v/cold",
+    visibility = ["//visibility:private"],
+)
+
+go_binary(
+    name = "cold",
```

Comment:
yeah, 'cold' is not that great name for this as @liranr23 pointed out below - how about virt-v2v-wrapper?